UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -854,6 +930,38 @@ public void RestartHeart()
 			}
 		}
 
+		[Server]
+		public void ApplyReagentsToSurface(ReagentMix Chemicals, BodyPartType bodyPartAim)
+		{
+
+			foreach (var reaction in ALLExternalMetabolismReactions)
```

Comment:
This is an `O(n)` operation. I think it's only called rarely, so it's fine. We should just keep it in mind.